tests/test_text_generation_example.py

```diff
@@ -218,6 +217,38 @@ def _test_text_generation(
     if "gemma" in model_name.lower():
         command += ["--use_flash_attention"]
 
+    if "mamba" in model_name.lower():
+        from huggingface_hub import hf_hub_download
+        import subprocess
+
+        cmd1 = subprocess.Popen(["pip", "list"], stdout=subprocess.PIPE)
+        cmd2 = subprocess.Popen(["grep", "habana-torch-plugin"], stdin=cmd1.stdout, stdout=subprocess.PIPE)
+        cmd1.stdout.close()
+        version_no, _ = cmd2.communicate() 
+
```

Comment:
Did you try just using check_optimum_habana_min_version?

Reply:
Done.